src/pages/StockConfiguration/UseCaseConfiguration/Sections.js

```diff
@@ -0,0 +1,82 @@
+import i18n from '@dhis2/d2-i18n'
+import { ButtonStrip, TabBar, Tab } from '@dhis2/ui'
+import findIndex from 'lodash/findIndex'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { Button } from '../../../components'
+import Details from './Details'
+import General from './General'
+import styles from './Sections.module.css'
+import Transactions from './Transactions'
+
+export const Sections = ({ settings, handleSettings, element }) => {
+    const [selectedTab, setSelectTab] = useState(0)
+
+    const TabItems = [
+        {
+            key: 'general-tab',
+            label: i18n.t('General'),
+            content: General,
+        },
+        {
+            key: 'details-tab',
+            label: i18n.t('Details'),
+            content: Details,
+        },
+        {
+            key: 'transactions-tab',
+            label: i18n.t('Transactions'),
+            content: Transactions,
+        },
+    ]
+
+    const transactionKey = findIndex(TabItems, { key: 'transactions-tab' })
+
+    const CurrentSection = TabItems[selectedTab].content
+
+    return (
+        <>
+            <div className={styles.tabs}>
+                <TabBar fixed>
+                    {TabItems.map(({ label }, index) => (
+                        <Tab
+                            key={index}
+                            onClick={() => setSelectTab(index)}
+                            selected={index === selectedTab}
+                            disabled={index !== selectedTab}
+                        >
+                            {label}
+                        </Tab>
+                    ))}
+                </TabBar>
+            </div>
+            <div>
+                <span className={styles.tabSectionTitle}>
+                    {TabItems[selectedTab].label}
+                </span>
+
+                <CurrentSection
+                    settings={settings}
+                    handleSettings={handleSettings}
+                    element={element}
+                />
+
+                <ButtonStrip end>
+                    {selectedTab !== transactionKey && (
+                        <Button
+                            small
+                            onClick={() => setSelectTab(selectedTab + 1)}
+                            title={i18n.t('Next')}
```

Comment:
Does it make sense to also add a `Prev` button?

Reply:
The user can only add more settings in this first app version. The second and the third tab depend on the program selected in the first tab. There is a "Next" button but not a "Prev" to prevent changing the program.

Reply:
Thanks for the suggestion. After talking about it, we are going with the Next and Prev button, so I'll add it in this PR.